### PR TITLE
Remove travis browserstack addon to fix HTTP error

### PR DESCRIPTION
**Pull request by jvanbruegge**
I saw that in master the setup is broken, we had the same issue in the cycle repo lately. This is the fix we used

### CODE REVIEW

**Comment by jvanbruegge** [comment]
@paldepind As you can see now the CI just does not pass because some tests fail is some browsers

**Comment by paldepind** [comment]
Thanks a lot. I really appreciate it :+1: I'll uncomment the browsers that fail.

I have a quick question which you may know the answer to. I'm trying to create a BrowserStack badge that we can add to the readme. Then BrowserStack will upgrade us to their open source plan. They write this in their instructions:

> To add status badges to a project, you need to give it a project name. If you haven’t specified a project name for your builds / sessions, add the capability project: <project_name> in your test scripts.

When I look at the current projects we only have one named "Untitled Project". Do you know how we can change the configuration such that we get a project named "Snabbdom"?

File: karma.conf.js [hunk 1]
```diff
@@ -2,8 +2,7 @@ const ci = !!process.env.CI;
 const watch = !!process.env.WATCH;
 const live = !!process.env.LIVE;
 
-const identifier = process.env.BROWSERSTACK_LOCAL_IDENTIFIER;
-const ip = process.env.IP_ADDR;
+const ip = 'bs-local.com';
```
Comment (paldepind) [inline karma.conf.js]: Why that ip?

Comment (jvanbruegge) [reply karma.conf.js]: That is a domain that browserstack owns and knows how to forward to the device. `localhost` or the old way do not work any more

**Comment by jvanbruegge** [comment]
> When I look at the current projects we only have one named "Untitled Project". Do you know how we can change the configuration such that we get a project named "Snabbdom"?

Oh, yeah, forgot that. Just add `name: 'Snabbdom'` to the `browserStack` key in `karma.conf.js`. For the disclaimer this is what we wrote: https://github.com/cyclejs/cyclejs#thanks

**Comment by jvanbruegge** [comment]
Oh, I saw that the name is already set, let me look for the other differences

**Comment by paldepind** [comment]
In case it helps. To see the projects I used the following command:

```
> curl -u "snabbdombot1:Wx2FUSehQLBf3haqHphd" https://api.browserstack.com/automate/projects.json
[{"id":817691,"name":"Untitled Project","group_id":967240,"user_id":3319678,"created_at":"2019-09-04T09:35:51.000Z","updated_at":"2019-09-04T10:20:44.000Z"}]
```

**Comment by jvanbruegge** [comment]
Maybe in the browserstack UI? I can not find differences to the cycle config

**Comment by paldepind** [comment]
That's odd. It looks good to me as well.

For the time being I've added just a Travis badge and the same disclaimer as used in the CycleJS repo (I hope that is okay).

I've emailed BrowserStack to give us the full plan.

Thank you so much for helping set this up and for continuing to fix things.

**Comment by jvanbruegge** [comment]
No problem, most of it was copy paste from cyclejs

**Pull request merged**